src/runtime/image.ts

```diff
@@ -56,7 +57,7 @@ export function createImage (context, { providers, defaultProvider, presets, int
     return presetMap[name]
   }
 
-  function image (source: string, modifiers: ImageModifiers, options: any = {}) {
+  function parseImage (source: string, modifiers: ImageModifiers, options: any = {}) {
```

Comment:
I would suggest naming `createImage` since we do more than parsing and basically creating an image object